### PR TITLE
#185 Ensure build job reports via email on failed merge to main

### PR DESCRIPTION
**Pull request by ndoschek**
Part of https://github.com/eclipse-emfcloud/emfcloud/issues/185

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -48,18 +48,18 @@ pipeline {
     options {
         buildDiscarder logRotator(numToKeepStr: '15')
     }
-    
+
     environment {
         YARN_CACHE_FOLDER = "${env.WORKSPACE}/yarn-cache"
         SPAWN_WRAP_SHIM_ROOT = "${env.WORKSPACE}"
+        EMAIL_TO = "ndoschek+eclipseci@eclipsesource.com, eneufeld+modelserver-ci@eclipsesource.com"
```
Comment (eneufeld) [inline Jenkinsfile:55]: why not use same name after the `+` ?

**Review by eneufeld — APPROVED** [review]
Except for the different build identifiers, this looks good to me.
I hope the JOB_NAME variable contains the repo name, too

File: Jenkinsfile (hunk 1 above)
Comment (ndoschek) [reply Jenkinsfile:55]: I'll update your email addresses to the same suffix as mine

**Comment by ndoschek** [comment]
Thanks @eneufeld , as discussed, I updated your mail addresses in the PRs to the same suffix as mine.

Regarding the JOB_NAME variable: yes, this contains the repo name.
The mail looks like this (here an example from the tree-editor):
```
Subject:
Build eclipse-emfcloud/theia-tree-editor/master (#19) FAILURE

Body:
Job: eclipse-emfcloud/theia-tree-editor/master
Build Number: 19
Build URL: https://ci.eclipse.org/emfcloud/job/eclipse-emfcloud/job/theia-tree-editor/job/master/19/
```

**Review by eneufeld — APPROVED** [review]
Thank you!

**Pull request merged**